lib/inspec/plugin/v2/plugin_types/cli.rb

```diff
@@ -1,5 +1,13 @@
 require 'inspec/base_cli'
 
+# Under some conditions (unit testing in particular) we may end
```

Comment:
I think we may have a merge conflict with: https://github.com/inspec/inspec/pull/3758/files#diff-0fdac3e925822c85c27f47879d14c973R3

I'm fine with either message but I prefer the one line approach over there. Up to you though. I have no strong opinion.

Reply:
AHA! Well, we found the PR in which I hit that issue, at least. We'll work out any conflict.